pkg/traces/servicegraphprocessor/processor.go

```diff
@@ -211,23 +264,80 @@ func (p *processor) ConsumeTraces(ctx context.Context, td pdata.Traces) error {
 		level.Error(p.logger).Log("msg", "failed consuming traces", "err", errs)
```

Comment:
instead of failing to consume traces maybe we should metric unprocessed spans but let the trace pipeline continue working.

Reply:
Agree, changed in eeb762f67d744e92104c8642ee43ca89f517ae8c